docs: Add CLI docs for 0.16.1 and 0.17.0 (#1300)
keptn/keptn.github.io: content/docs/0.17.x/reference/cli/commands/keptn_create_project.md

> The shipyard file describes the used stages. These stages are defined by name, as well as their task sequences.

Reviewer: ```suggestion
The [shipyard](https://keptn.sh/docs/0.17.x/reference/files/shipyard/) file describes the stages that are used. These stages are defined by name, as well as their task sequences.
```

By executing the *create project* command, Keptn initializes an internal Git repository that is used to maintain all project-related resources. 

> To upstream this internal Git repository to a remote repository, the remote URL (*--git-remote-url*) is required
> together with private key (*--git-private-key*) or access token (*--git-token*). The Git user (*--git-user*) can be specified if the repository allows it. 

Reviewer: ```suggestion
You must specify the remote URL (*--git-remote-url*)
and the private key (*--git-private-key*) or access token (*--git-token*) in order to upstream this internal Git repository to a remote respository. The Git user (*--git-user*) can be specified if the repository allows it. 
```

> For using proxy please specify proxy IP address together with port (*--git-proxy-url*) and

Reviewer: ```suggestion
If using a proxy, please specify the proxy IP address, port (*--git-proxy-url*) and
```

> used scheme (*--git-proxy-scheme=*) to connect to proxy. Please be aware that authentication with public/private key and via proxy is 

Reviewer: ```suggestion
and scheme (*--git-proxy-scheme=*) that is used to connect to the proxy. Please be aware that authentication with public/private key and via proxy is 
```
